### ISSUE TITLE
Role Label Order

### ISSUE THREAD

**Issue by ShendMikullovci** [serge-web/serge#1988]
Deleting Mar-1 and adding it back puts Mar-1 at the bottom of the list in the admin page and in the welcome screen for the game:

![Mar-1 bottom](https://user-images.githubusercontent.com/85491794/198874590-b7f27a20-b98f-4891-8afd-21183a0bdc98.png)
![Mar-1 bottom game screen](https://user-images.githubusercontent.com/85491794/198874594-a5251328-a462-47c1-8753-06133dbc381f.png)

I suggest we order these alphabetically, that way they all appear ordered and make it easier for GC to look for a certain role.

**Comment by IanMayo** [comment]
Yes @ShendMikullovci .  They are currently stored and displayed in the order they were generated.  

It would be neater to display them in alphabetical order.
